ckanext/ldap/controllers/user.py

```diff
@@ -321,14 +329,15 @@ def _ldap_search(cnx, filter_str, attributes, non_unique=u'raise'):
 
     '''
     try:
-        res = cnx.search_s(config[u'ckanext.ldap.base_dn'], ldap.SCOPE_SUBTREE,
+        res = cnx.search_s(base_dn_str, ldap.SCOPE_SUBTREE,
                            filterstr=filter_str, attrlist=attributes)
+        '''res = cnx.search_s(None, ldap.SCOPE_SUBTREE, filterstr=filter_str, attrlist=attributes)
+        '''
```

Comment:
can commented out code string thing this be removed please?

Reply:
Sure it can. Should not have been committed.